src/Gadfly.jl

```diff
@@ -46,6 +46,11 @@ abstract GeometryElement    <: Element
 abstract GuideElement       <: Element
 abstract StatisticElement   <: Element
 
+# The names stashed on the Aesthetics object were retrieved with .names
+# in v0.3. That moves to fieldnames in v0.4. When v0.3 support is ended
+# you can delete this line
+if VERSION < v"0.4-dev" const fieldnames = names end
```

Comment:
I think Compat should just handle this for you without needing this line

https://github.com/JuliaLang/Compat.jl/blob/2c91b7a12d1f206f06c902a2b687cc2587f7f614/src/Compat.jl#L195-L205